netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
```

Comment:
Can we add a redirect for the [white paper](https://zeppelinos.org/zeppelin_os_whitepaper.pdf) as well? Not sure where it will be hosted in the new site, though.

Reply:
done.
It will go to https://openzeppelin.com/assets/zeppelin_os_whitepaper.pdf